azure-pipelines.yml

```diff
@@ -0,0 +1,74 @@
+trigger:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - azure-pipelines.yml
+    - 'Fakebook.Profile'
+
+pr:
+- 'main'
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+stages:
+- stage: build
+  jobs:
+  - job: build
+    variables:
+      sdkVersion: '5.x'
+      buildConfiguration: 'Release'
+      targetProject: 'Fakebook.Profile/Fakebook.Profile.RestApi'
+      targetSolution: 'Fakebook.Profile'
+  
+    steps:
+    - task: UseDotNet@2
+      displayName: dotnet sdk 2.x
+      inputs:
+        packageType: 'sdk'
+        version: '2.x'
+
+    - task: SonarCloudPrepare@1
+      inputs:
+        SonarCloud: 'SonarCloud Token'
+        organization: '2011-fakebook-project3'
+        scannerMode: 'MSBuild'
+        projectKey: '2011-fakebook-project3_profile'
+        projectName: 'profile'
+        extraProperties: |
+          sonar.exclusions=**/lib/**
```

Comment:
no purpose to this exclusions line since this isnt mvc and there is no such folder